.gitignore

```diff
@@ -36,7 +36,7 @@ report.json
 /npm/path.txt
 
 # Electron Fiddle-Specific
-/static/contributors.json
+# /static/contributors.json
```

Comment:
Let's revert to just doing say `echo "[]" > static/contributors.json` in the jobs before running tests - should work for all platforms. Otherwise we'll end up in the annoying situation of this file accidentally getting committed in PRs, etc.

Reply:
Yeah, totally agreed, this was a dirty debugging tactic, should not be left this way 👍